lang/java/tools/pom.xml

```diff
@@ -145,7 +145,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-core</artifactId>
```

Comment:
you can use the dependency into the specific profile : see ./lang/java/mapred/pom.xml
the use of hadoop-core and hadoop-client regarding the profile hadoop1 or hadoop2